Dockerfile

```diff
@@ -26,6 +26,9 @@ vim \
 git \
 cron \
 mysql-client \
+<<<<<<< HEAD
```

Comment:
This looks like a merge conflict gone wrong?

Reply:
> This looks like a merge conflict gone wrong?

Let me fix that.